### DISCUSSION TITLE
reporting an issue is so complicated.

### DISCUSSION THREAD

**Discussion by Knvgpt** [will-stone/browserosaurus#639] [General]
All I wanna say it randomly crashes, and then when I have to restart it, it asks for coffee, so maybe that's the issue.

**Comment by will-stone** [comment]
Many many hours of work have gone into making, refining, and maintaining this free and open source app. Not spending the 5 minutes required to fill out a well formed issue report will win you zero support time. Please be considerate of others. Thanks for using Browserosaurus ☺️
